### ISSUE TITLE
60 Fps CAP

### ISSUE THREAD

**Issue by DrCoomerPhD** [sultim-t/xash-rt#103]
IDK if its required but, the games capped at 60 fps and cant go higher, thanks.

**Comment by Dennis50300** [comment]
it brings nothing really useful to get higher
get understanding the math of PC Hardware and Peripherals with it ;-)

https://humanbenchmark.com/
here you can test, without CPU oder GPU -limit you doesnt have a problem with 60HZ/FPS that you can the 180ms reactionstime without a problem that for us human are possible

**Comment by chaosophy** [comment]
Have you tried the latest 1.0.5 release?

From the update log...

Vsync option should work correctly now
Frame limiters are uncapped by default (https://github.com/sultim-t/xash-rt/issues/25) thanks to [a1batross](https://github.com/a1batross)

**Comment by TheRealDadbeard** [comment]
> it brings nothing really useful to get higher get understanding the math of PC Hardware and Peripherals with it ;-)
> 
> https://humanbenchmark.com/ here you can test, without CPU oder GPU -limit you doesnt have a problem with 60HZ/FPS that you can the 180ms reactionstime without a problem that for us human are possible

you understand nothing at all lol

**Comment by JDFind** [comment]
@chaosophy Somehow framerate is still capped at 60fps, using 1.0.5, 7900 XTX, driver 23.2.2.

**Comment by kheperaptah** [comment]
Yup, having fps caps except its at 57fps not 60 while using 1.0.5

**Comment by Dennis50300** [comment]
> > it brings nothing really useful to get higher get understanding the math of PC Hardware and Peripherals with it ;-)
> > https://humanbenchmark.com/ here you can test, without CPU oder GPU -limit you doesnt have a problem with 60HZ/FPS that you can the 180ms reactionstime without a problem that for us human are possible
> 
> you understand nothing at all lol

crystals...
if you can outsmart physics and prove it, please let us know with the technology from a distant galaxy...
you are the one that understand nothing :-)

you an advertising victim, sry thats the reality

**Comment by TheRealDadbeard** [comment]
> > > it brings nothing really useful to get higher get understanding the math of PC Hardware and Peripherals with it ;-)
> > > https://humanbenchmark.com/ here you can test, without CPU oder GPU -limit you doesnt have a problem with 60HZ/FPS that you can the 180ms reactionstime without a problem that for us human are possible
> > 
> > 
> > you understand nothing at all lol
> 
> crystals... if you can outsmart physics and prove it, please let us know with the technology from a distant galaxy... you are the one that understand nothing :-)
> 
> you an advertising victim, sry thats the reality

You're really funny. Thanks for the laughs

**Comment by phant** [comment]
> IDK if its required but, the games capped at 60 fps and cant go higher, thanks.


> Yup, having fps caps except its at 57fps not 60 while using 1.0.5

Did you try lower resolution or enable FSR/DLSS ?

I have ~60fps with no upscalers at 4k, and [100-110 with DLSS/FSR quality](https://www.youtube.com/watch?v=IlT9lhkFN34)

**Comment by Riker6** [comment]
So I noticed that if I use the command cl_showfps 1, it shows I am getting 120 FPS with FSR Performance but it is still only around 60 FPS as I can see with my eyes and feel of the mouse it is only 60, even on 1.0.5.

When I check 'No server cull", boom, I can visually see I am getting 120FPS!

Perhaps there is something that needs to be done, like a tickrate or cl_updaterate change? I wonder if the way the server/client usage is occurring ends up making our movement capped at 60 (when strafing I can very easily see when 60 or 120FPS since I have an OLED display). I am just spitballing here...

**Comment by JDFind** [comment]
> @chaosophy Somehow framerate is still capped at 60fps, using 1.0.5, 7900 XTX, driver 23.2.2.

I can confirm this happens with or without FSR (the GPU just gets less utilization, but no FPS boost), with or without "No server cull" option. 
Also set `gl_vsync "0"`, `sleeptime 0`, `vid_displayfrequency "200"`, `fps_max "300"` variables in config.cfg, with not effect.
Screen resolution 1440p, 60Hz monitor frequency.

**Comment by Riker6** [comment]
The only solution to allow greater than 60 FPS to actually be displayed (since the FPS counter in game was not reliable) was to have "No server cull" option checked.

I took slomo video to show how FPS counter doesn't capture the ~60FPS cap. Both videos let you see the 120FPS counter on the top right. As you can see, the second video where full 120FPS is actually displaying is the smoother one.

["No server cull" unchecked (Apparent 60FPS cap)](https://youtu.be/uPfEahpOaZc)

["No server cull" checked (Full 120FPS actually displaying)](https://youtu.be/O5PYwN3h9GU)

**Comment by bjscaggles** [comment]
> it brings nothing really useful to get higher get understanding the math of PC Hardware and Peripherals with it ;-)
> 
> https://humanbenchmark.com/ here you can test, without CPU oder GPU -limit you doesnt have a problem with 60HZ/FPS that you can the 180ms reactionstime without a problem that for us human are possible

Higher fps in video games = Images you see are the closest possible to what is currently happening within the game engine. It has almost nothing to due with the eyes being able to detect the frames and everything to do with input and output latency.


Also, show me a 144hz monitor at 144 fps vs 60hz monitor at 60fps and I will be able to choose the correct one every single time. This has been proven many times over.

**Comment by SlightlyPeckish** [comment]
My game is capped at 56fps no matter what options I change if I use DLSS or FSR or Server Cull or v-sync it doesn't seem to make a difference 
My monitor is 144hz
CPU: AMD Ryzen 9 5950X 16-Core Processor 
GPU: NVIDIA GeForce RTX 4090
